client/landing/gutenboarding/onboarding-block/acquire-intent/vertical-select/style.scss

```diff
@@ -48,6 +33,7 @@
 	color: var( --studio-gray-5 );
 	position: absolute;
 	left: 0;
+	width: 400px;
```

Comment:
This `400px` is stolen from [here](https://github.com/Automattic/wp-calypso/pull/40808/files#diff-aa716e317d5a3920b02ab59f11dd252aL21). Not a regression, but certainly not a progression either.